src/config/docker.md

```diff
@@ -0,0 +1,96 @@
+# Docker
+
+Docker is a platform of products to build isolated containers for programs using
+visualization.
+
+## Getting Started with Docker.
+
+After installing the 'docker' package (and optionally, the `docker-compose`
+package), enable the `docker` service:
+
+```
+# ln -s /etc/sv/docker /var/service
+```
+
+and add your user to the `docker` group:
+
+```
+# usermod -aG docker $USER
+```
+
+You will know you have properly installed docker if you can run it in the
+terminal with the `docker` command. Issuing this command will result in a long
+list of text displayed to users about docker and how it can be used. It will
+result in text being displayed like the following:
+
+```
+$ docker
+
+Usage:  docker [OPTIONS] COMMAND
+
+A self-sufficient runtime for containers
+
+Options:
+  ...
+      --tlsverify          Use TLS and verify the remote
+  -v, --version            Print version information and quit
+
+Management Commands:
+  ...
+  trust       Manage trust on Docker images
+  volume      Manage volumes
+
+Commands:
+  ...
+  version     Show the Docker version information
+  wait        Block until one or more containers stop, then print their exit codes
+
+Run 'docker COMMAND --help' for more information on a command.
+```
```

Comment:
I'd prefer we not include command output, makes the page more crowded for little benefit.